src/aggregator.ts

```diff
@@ -1,5 +1,5 @@
 import { Address, log, store } from "@graphprotocol/graph-ts";
-import { AggregatorUpdated } from "../generated/Aggregator/AggregatorContract";
+import { AggregatorUpdated } from "../generated/Aggregator-0_2_0/AggregatorContract_0_2_0";
```

Comment:
I don't understand this? isn't it the same entity for both contracts?

Reply:
It's the generated file paths